app/services/workflow_reporter.rb

```diff
@@ -17,9 +18,14 @@ def self.report_error(druid, process_name, error_message)
     else
       Rails.logger.warn(NO_WORKFLOW_HOOKUP)
     end
+  rescue Dor::WorkflowException => e
+    raise unless e.message.match?(MISSING_WF_REGEX)
```

Comment:
This seems a little fragile and ideally this gets a follow up PR in the workflow client.

Reply:
See issue sul-dlss/dor-workflow-client/issues/135